script/validate

```diff
@@ -2,19 +2,40 @@
 
 import json
 import sys
+from collections import Counter
 
 import jsonschema
 import yaml
 from jsonschema import validate
 
+
+def extract_urns(data, urns=None):
+    if urns is None:
+        urns = []
+
+    if isinstance(data, dict):
+        for key, value in data.items():
+            if key == "urn":
+                urns.append(value)
+            else:
+                extract_urns(value, urns)
+    elif isinstance(data, list):
+        for item in data:
+            extract_urns(item, urns)
```

Comment:
Maybe add and else clause writing to stderr when an unexpected type of data is found, or throw an exception? Otherwise, maybe add a comment explaining why we do not provide an else statement.

Reply:
Hmm, what else could it be that we want to act differently on? The only other case is an atomic value (int, string, float). In which case we're done and we simply return the urns thus far. What am I missing?

Reply:
I'm merging this PR, if you'd like to see change I'll followup with a PR